src/SourceBuild/content/repo-projects/fsharp.proj

```diff
@@ -19,8 +19,13 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <RepositoryReference Include="arcade" />  
+    <RepositoryReference Include="runtime" />
```

Comment:
fsharp doesn't have a live dependency on runtime in MSFT modes..this should move into source-only.

Reply:
Fixed in https://github.com/dotnet/installer/pull/18557/commits/a0117ac75d9ea8a2a65d52800c4735b7dc08b0e8